src/documentation/programming-guide.md

```diff
@@ -334,6 +439,16 @@ PCollection&lt;Integer&gt; wordLengths = words.apply(
       .withOutputType(new TypeDescriptor&lt;Integer&gt;() {});
 ```
 
+```python
+# The input PCollection of Strings.
+words = ...
+
+# Apply a Map with a ambda function to the PCollection words.
+# Save the result as the PCollection wordLengths.
+wordLengths = words | beam.Map(lambda x: [len(x)])
```

Comment:
`Map` returns only one element since it has to be a 1-1 mapping

```
# The input PCollection of string.
words = ...

# Apply a Map with a lambda function to the PCollection words.
# Save the result as the PCollection word_lengths.
word_lengths = words | beam.Map(lambda x: len(x))
```